src/cgtop/cgtop.c

```diff
@@ -605,7 +614,7 @@ static void display(Hashmap *a) {
         for (j = 0; j < n; j++) {
                 unsigned cputlen, pathtlen;
 
-                format_timespan(buffer, sizeof(buffer), (usec_t) (array[j]->cpu_usage / NSEC_PER_USEC), 0);
+                maybe_format_timespan(buffer, sizeof(buffer), (usec_t) (array[j]->cpu_usage / NSEC_PER_USEC), 0);
```

Comment:
hmm, could you update the sizing of buffer[] to also include DECIMAL_STR_MAX(uint64_t) in its maximum detection? after all we should make sure that the buffer fits the 64bit value too

Reply:
there's no MAX4 macro yet , should I define one or just type `MAX(MAX3(), z)`

Reply:
define it